cyences_app_for_splunk/default/data/ui/views/cs_vulnerability.xml

```diff
@@ -0,0 +1,196 @@
+| rename dest_id as asset_id 
+| lookup cs_all_assets asset_id OUTPUT asset_ip, asset_hostname, asset_mac_address, os, state 
+| eval ip = split(asset_ip, "~~"), hostname = split(asset_hostname, "~~"), mac_address = split(asset_mac_address, "~~") 
+| fillnull value="Not Present" ip, hostname, mac_address
+| table asset_id, ip, hostname, mac_address, os, total_vuln, fixed_vuln, active_vuln, critical_active_vuln, high_active_vuln, fixed_vuln, medium_active_vuln, low_active_vuln, info_active_vuln, last_scan_time 
+| sort - active_vuln</query>
+    <earliest>$timerange.earliest$</earliest>
+    <latest>$timerange.latest$</latest>
+  </search>
+  <search id="vul_list_search">
+    <query>| tstats `cs_summariesonly_cyences_vulnerabilities` count, dc(Vulnerabilities.dest_id) as no_of_hosts, first(Vulnerabilities.signature) as signature, first(Vulnerabilities.category) as category, first(Vulnerabilities.severity) as severity, first(Vulnerabilities.status) as status, first(Vulnerabilities.type) as type, first(Vulnerabilities.cve) as cve, first(Vulnerabilities.protocol) as protocol, first(Vulnerabilities.port) as port, first(Vulnerabilities.first_found) as first_found, first(Vulnerabilities.last_found) as last_found FROM datamodel=Cyences_Vulnerabilities where Vulnerabilities.vendor_product="$product$" Vulnerabilities.dest_id="$asset_id$" by Vulnerabilities.vul_id 
+| `drop_dm_object_name(Vulnerabilities)` 
+| fillnull value="Not Present" cve, signature
+| eval sort_field=case(severity="critical",1, severity="high",2, severity="medium",3, severity="low",4, severity="informational",5, 1=1, 6) 
+| sort sort_field
+| table vul_id signature category no_of_hosts severity status cve protocol port first_found last_found</query>
+    <earliest>$timerange.earliest$</earliest>
+    <latest>$timerange.latest$</latest>
+  </search>
+  <fieldset submitButton="false">
+    <input type="time" token="timerange" searchWhenChanged="true">
+      <label></label>
+      <default>
+        <earliest>-7d@h</earliest>
```

Comment:
Shall we just have a longer default time range? Something like 3 months or 1 year? So we can cover as many as the latest vulnerabilities?

Reply:
And have a separate time range for all time charts maybe. Just in case they want to see the last 7 days, users don't need to update the dashboard time range.